### PR TITLE
Upgrade play version

### PR DESCRIPTION
**Pull request by RikSherman**

### CODE REVIEW

File: app/uk/gov/hmrc/platformstatusfrontend/config/ErrorHandler.scala [hunk 1]
```diff
@@ -21,11 +21,11 @@ import javax.inject.{Inject, Singleton}
 import play.api.i18n.MessagesApi
 import play.api.mvc.Request
 import play.twirl.api.Html
-import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
 import uk.gov.hmrc.platformstatusfrontend.views.html.error_template
```
Comment (colin-lamed) [inline app/uk/gov/hmrc/platformstatusfrontend/config/ErrorHandler.scala:24]: Can we rename the html templates to use CamelCase - which is compatible with Scala type naming?

File: app/uk/gov/hmrc/platformstatusfrontend/controllers/CodeController.scala [hunk 2]
```diff
@@ -23,15 +23,15 @@ import play.api.data.Forms._
 import play.api.mvc._
 import uk.gov.hmrc.platformstatusfrontend.config.AppConfig
 import uk.gov.hmrc.platformstatusfrontend.services.{MemoryHog, StatusChecker}
-import uk.gov.hmrc.platformstatusfrontend.views
-import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import uk.gov.hmrc.platformstatusfrontend.views.html.{code, codeResponse}
```
Comment (colin-lamed) [inline app/uk/gov/hmrc/platformstatusfrontend/controllers/CodeController.scala:26]: Rename templates here too?

File: build.sbt [hunk 3]
```diff
@@ -1,14 +1,14 @@
 import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
-import uk.gov.hmrc.SbtArtifactory
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
 val appName = "platform-status-frontend"
 
 lazy val microservice = Project(appName, file("."))
-  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
+  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtDistributablesPlugin)
```
Comment (colin-lamed) [inline build.sbt:7]: `SbtAutoBuildPlugin` can also be removed

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -7,21 +7,20 @@ object AppDependencies {
...
+    "org.mongodb.scala"       %% "mongo-scala-driver"          % "2.6.0"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-play-26"        % "0.42.0"              % Test classifier "tests",
-    "org.scalatest"           %% "scalatest"                % "3.0.8"               % Test,
-    "org.scalacheck"          %% "scalacheck"               % "1.14.3"              % Test,
-    "org.jsoup"               %  "jsoup"                    % "1.10.2"              % Test,
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"   % "5.14.0"              % Test,
+    "com.vladsch.flexmark"    %  "flexmark-profile-pegdown" % "0.62.2"              % Test,
```
Comment (colin-lamed) [inline project/AppDependencies.scala:18]: do we need `flexmark-profile-pegdown`? `bootstrap-test-play-28` already adds flexmark - which I suspect is enough

File: project/AppDependencies.scala [hunk 5]
```diff
@@ -7,21 +7,20 @@ object AppDependencies {
...
+    "org.scalacheck"          %% "scalacheck"               % "1.15.0"              % Test,
+    "org.scalatestplus"       %% "scalacheck-1-15"          % "3.2.10.0"            % Test,
+    "org.jsoup"               %  "jsoup"                    % "1.14.3"              % Test,
     "com.typesafe.play"       %% "play-test"                % current               % Test,
```
Comment (colin-lamed) [inline project/AppDependencies.scala:22]: `play-test` can be removed - it is provided by `bootstrap-test-play-28`

File: project/plugins.sbt [hunk 6]
```diff
@@ -1,17 +1,11 @@
...
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.19.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "0.19.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.5.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.6.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.0.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.20")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.20")
```
Comment (colin-lamed) [inline project/plugins.sbt:11]: should this be `2.8.8` now?

Comment (RikSherman) [reply project/plugins.sbt:11]: Yes d'oh

**Pull request merged**